.circleci/config.yml

```diff
@@ -129,6 +129,20 @@ jobs:
             - -pkg2-v20220303-{{ checksum "Makefile.envs" }}
             - -pkg2-v20220303
 
+      - run:
+          name: install rust
+          command: |
+            wget https://sh.rustup.rs -O /rustup.sh
+            sh /rustup.sh -y
+            source $HOME/.cargo/env
+            # We need nightly because we need to build the standard library.
+            # -Zbuild-std only works on nightly:
+            # https://doc.rust-lang.org/cargo/reference/unstable.html#requirements
+
+            # Make sure nightly-2022-05-28 matches the version in Makefile.envs
+            rustup toolchain install nightly-2022-05-28
+            rustup component add rust-src --toolchain nightly-2022-05-28
```

Comment:
I think we will have to add this rust installation step in our Makeflie (e.g. `make rust`) so that people can build the cryptography package locally.